### PR TITLE
Added copy and a wee bit of code

### PR DESCRIPTION
**Pull request by maggiemelo**
Hi, all. I added copy to the WTM page. I also was feeling pretty fierce and added a wee bit of code. Hope it works out okay!

### CODE REVIEW

File: pages/womentechmakers.html [hunk 1]
```diff
@@ -9,10 +9,28 @@
...
+  <li>Mentoring and on-site tech support during project development</li>
+  <li>Professional development opportunities and community partnership occasions</li>
+  <li>Amazing swag, prizes, and food</li>
+</ul>
+
+<p> Don't miss out! Click <a href="http://www.meetup.com/Google-Developer-Group-Tucson/events/225462550/?a=socialmedia">here</a> to register for the event--space is limited. Have more questions? Check out our FAQ below. </p>
+
+</p>
+
+</p>
```
Comment (dmsnell) [inline pages/womentechmakers.html]: I'm wondering what purpose these tags hold here, are you trying to add some additional space before the FAQ? If so, this a wonderful case to apply a little CSS. Ping me if you would like to know more!

File: pages/womentechmakers.html [hunk 2]
```diff
@@ -9,10 +9,28 @@
...
+  <p>Google Development Group, Code for Tucson, and UA Libraries are proud to present the <b> Women Techmakers Tucson Hackathon </b> on November 14-15 at the University of Arizona's Science and Engineering Library.
+
+<p> Join us for a weekend of food, fun, learning, and hacking! This free event provides an opportunity for women of all levels of technical expertise to collaboratively develop tech-driven projects over the course of two days. All women are encouraged to attend--no computer programming experience required! </p>
+
+<p> <b> Event highlights: </b> </p>
```
Comment (dmsnell) [inline pages/womentechmakers.html]: I'm not sure if we have this as an example elsewhere, which would be a poor example, but most developers prefer not to use `<b />` and `<i />` anymore, favoring the for _semantically_ opinionated `<strong />` and `<em />` respectively.

"strong" is bold by default
"_em_ phasis" is italicized by default

**Comment by dmsnell** [comment]
@maggiemelo hi ho. I added just a couple comments for you to look at. Be fierce, be bold!

File: pages/womentechmakers.html (hunk 2 above)
Comment (maggiemelo) [inline pages/womentechmakers.html]: Ah, noted. Thanks for that!

**Comment by dmsnell** [comment]
Looks good to me!

:shipit: (this is the "Ship it" squirrel, meaning you should feel confident to hit the "Merge" button)

**Comment by maggiemelo** [comment]
Awesome! Thanks again!

On Thursday, October 15, 2015, Dennis Snell <notifications@github.com>
wrote:

> Looks good to me!
>
> [image: :shipit:] (this is the "Ship it" squirrel, meaning you should
> feel confident to hit the "Merge" button)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CodeForTucson/codefortucson-site/pull/53#issuecomment-148521832>
> .
>


-- 
*Marijel (Maggie) Melo, MA*
PhD Student and Graduate Associate in Teaching
Rhetoric, Composition, and the Teaching of English
Lead Organizer, *Women TechMakers Tucson Hackathon
<http://www.meetup.com/Google-Developer-Group-Tucson/events/225462550/>*
Coordinator, *Innovation Space (iSpace)
<http://www.library.arizona.edu/visit/spaces/ispace>*
The University of Arizona

Let's connect. LinkedIn profile:
www.linkedin.com/pub/marijel-maggie-melo/12/2a7/131/

**Pull request merged**